docs/modding-and-development/development/reviewing.md

```diff
@@ -54,7 +57,7 @@ In your final comment make sure to mention what you did, e.g. which aspects of t
 After leaving your comment, select the type of feedback.
 
 * **Comment**: General remarks, use if neither of the later two applies.
-* **Approve**: You found no issues when looking at the code or during testing.
+* **Approve**: You found no issues when looking at the code or during testing. **Only approve a PR if you'd be willing to maintain the code change in question. Merging a pull request is essentially handing off responsibility of the changed code from the PR contributor to the codebase maintainer!**
```

Comment:
> You found no issues when looking at the code or during testing.

It should be "[...] at the code *AND* during testing", since I can't imagine reviewers hitting the approve button without actually running the proposed code.

Reply:
> I can't imagine reviewers hitting the approve button without actually running the proposed code.

Oh sweet summer child if only you knew the things I've seen :P

Reply:
> if only you knew the things I've seen

I for sure hope my research colleagues are not reading my words 🙄